python/bolt12.py

```diff
@@ -111,24 +117,174 @@ def simple_bech32_decode(bech32str: str) -> Tuple[str, bytes]:
         return None, None
 
     ret5 = bytes()
-    for c in bech32str[sep+1:]:
+    for c in bech32str[sep + 1:]:
         pos = bech32.CHARSET.find(c)
         if pos == -1:
             return None, None
         ret5 += bytes([pos])
-    return hrp, bytes(bech32.convertbits(ret5, 5, 8, False))
+
+    intarr = bech32.convertbits(ret5, 5, 8, False)
+    if intarr is None:
```

Comment:
Okay will it ever be None? I haven't handled it in JS..

Reply:
No, it could only return None if we gave it a value which wasn't in range 0-31. Obviously we don't, since we just decoded it.  But mypy complains it can be None, so I handled it :)